data/json/monsters/mammal.json

```diff
@@ -704,15 +704,44 @@
     "vision_night": 10,
     "path_settings": { "max_dist": 10 },
     "special_attacks": [ [ "scratch", 10 ], { "type": "leap", "cooldown": 5, "max_range": 5 } ],
-    "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
+    "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT", "PLAYER_CLOSE", "FRIEND_ATTACKED", "PLAYER_NEAR_BABY" ],
     "fear_triggers": [ "SOUND" ],
     "zombify_into": "mon_zougar",
     "harvest": "cougar_with_skull",
     "dissect": "dissect_feline_sample_small",
     "families": [ "prof_intro_biology", "prof_physiology" ],
+    "reproduction": { "baby_monster": "mon_cougar_kitten", "baby_count": 6, "baby_timer": 92 },
+    "//": "Cougars can breed all year, but mainly do so between January and August, they can have between 1 to 6 kittens at a time.",
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN" ],
     "flags": [ "SEES", "HEARS", "GOODHEARING", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "HIT_AND_RUN", "KEENNOSE" ],
     "armor": { "bash": 1 }
   },
+  {
+    "id": "mon_cougar_kitten",
+    "type": "MONSTER",
+    "copy-from": "mon_cougar",
+    "looks_like": "mon_cat_kitten",
+    "name": { "str": "cougar kitten" },
+    "description": "A cuddly and defenseless cougar kitten, meowing at its surroundings in search of protection.  Be careful, these kittens grow quite big and ferocious in just a few months.",
```

Comment:
I'd be careful with calling it "cuddly" in the description when it's not tameable in vanilla. On the offchance someone finally reads a monster description they may get the wrong idea here.

Reply:
Good point! I will change it later after testing then.